Header header

# Set of VSGraphsWallData.msg
VSGraphsWallData[] walls